BullsEye/ViewController.swift

```diff
@@ -0,0 +1,123 @@
+//
+//  ViewController.swift
+//  BullsEye
+//
+//  Created by Tim Lauter on 10/31/16.
+//  Copyright © 2016 timlauter. All rights reserved.
+//
+
+import UIKit
+import QuartzCore
+
+class ViewController: UIViewController {
+    @IBOutlet weak var slider: UISlider!
+    @IBOutlet weak var targetLabel: UILabel!
+    @IBOutlet weak var scoreLabel: UILabel!
+    @IBOutlet weak var roundLabel: UILabel!
+    
+    @IBAction func startOver() {
+        startNewGame()
+        updateLabels()
+        
+        let transition = CATransition()
+        transition.type = kCATransitionFade
+        transition.duration = 0.4
+        transition.timingFunction = CAMediaTimingFunction(name: kCAMediaTimingFunctionEaseOut)
+        view.layer.add(transition, forKey: nil)
+    }
+    
+    var currentValue: Int = 0
+    var targetValue: Int = 0
+    var score: Int = 0
+    var round: Int = 0
```

Comment:
to take this project to the next level, abstract out the currentValue, targetValue, score, and round into its own model  struct in a separate file, and use that model to populate the UI elements in this view controller.  Ideally, a view controller should only manage how and when views appear and transitions between other view controllers.